app/models/user.rb

```diff
@@ -58,8 +58,8 @@ def as_json(_options = {})
       admin: admin }
     if (_options[:follows])
       json['follows'] = {
-        topics: following.where(muted: false, followed_type: 'Topic').to_a.map(&:followed_id),
-        maps: following.where(muted: false, followed_type: 'Map').to_a.map(&:followed_id)
+        topics: following.active.where(followed_type: 'Topic').to_a.map(&:followed_id),
+        maps: following.active.where(followed_type: 'Map').to_a.map(&:followed_id)
```

Comment:
do you think active is better, or should we use the more explicit but awkward `unmuted`?

Reply:
I think active is ok